### ISSUE TITLE
seems that property "-webkit-font-smoothing" is not working

### ISSUE THREAD

**Issue by axten** [kutsan/stylelint-config-clean-order#34]
Hello!
seems that   `-webkit-font-smoothing` property is not ordered correctly with the other typo props but at the end.

**Comment by axten** [comment]
according to the readme at https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-order/README.md
```
...
Where font-smoothing is the unprefixed version of proprietary browser property -webkit-font-smoothing.
...
```
the config should just contain unprefixed variants in the group definitions.

**Comment by kutsan** [comment]
Hmm, thanks for reporting. I published a fix with [v5.4.2](https://github.com/kutsan/stylelint-config-clean-order/releases/tag/v5.4.2). Please let me know if you encounter any issues.

**Comment by axten** [comment]
now it's fixed - thanks for fast reaction!
